### ISSUE TITLE
Camo large ALICE pack invisible on female characters and green on male characters

### ISSUE THREAD

**Issue by 90sVenomFan** [sharkster91/SMUI#59]
The camo/woodland variant of the large ALICE Pack (or ALICE Pack as it's called in-game) is invisible while worn on female characters. I know this has been an ongoing problem as I've seen other people report the same thing, but I think it may also be happening due to an incompatibility with Authentic Z's backpacks+ mod because it's the only other mod I have active that changes the vanilla ALICE Pack.
However, I do have SMUI loaded after Authentic Z, so I am not entirely sure if that is what's causing the problem. I might check sometime in the future and update this when I have the chance.
Here's how it looks on female characters
![20221228165229_1](https://user-images.githubusercontent.com/121587802/209891606-dc2d61a1-99f7-4a22-a6bb-b1c9b1bdd5f8.jpg)
and here's how it looks on male characters (this is the same bag btw)
![20221228182705_1](https://user-images.githubusercontent.com/121587802/209891868-87f2ae74-d8e1-4265-83bd-9e4bb25095a4.jpg)

**Comment by 90sVenomFan** [comment]
Also sorry if this report isn't thorough enough, it's my first time reporting an issue with a mod. Any advice and tips on how to improve are greatly appreciated!

**Comment by 90sVenomFan** [comment]
Update: I've went through the mod's files and noticed a few oddities regarding the ALICE pack.
Firstly; there is no F_MediumALICE, which would indicate there's a female model. Instead the script connects to the male model for both the male and female versions, as seen here:
![ALICE modded](https://user-images.githubusercontent.com/121587802/209900370-c0d11794-f560-42e1-bc29-8f4306ef9375.png)
Second and lastly; the name of the woodland/camo variant has ALICE in mostly lower case letters, so it's spelled Alice, which may be causing the texture problem on the male model. I tried renaming it to be more inline with what's stated in the code, but it just reverts to the lower case form seen here, compared to the regular version:
![modded textures](https://user-images.githubusercontent.com/121587802/209900628-804b56f4-9d86-47f0-8eb9-2ab75f964796.png)
